examples/copper/kornia_app.ron

```diff
@@ -0,0 +1,41 @@
+(
+    tasks: [
+        (
+            id: "cam0",
+            type: "copper::tasks::video_capture::VideoCapture",
```

Comment:
We have a v4l src in copper, maybe reuse it and if we miss stuff add it back into it?
We also have a gstreamer source now if this is easier too

Reply:
I'd prefer to use the kornia apis, that's the main purpose of the example to show libraries integration

Reply:
if there's any performance issue that would help also from our side to iterate on bug fixing